ruby/vampires.rb

```diff
@@ -0,0 +1,56 @@
+puts "How many employees you will be processed?"
+num_empl = gets.chomp.to_i 
+
+
+i=0
+until i == num_empl
+
+	puts "What is your name?"
+	name = gets.chomp
+	name == "Drake Cula" || name == "Tu Fang"? name = true : name = false
```

Comment:
A cleaner way to write somethign like this could be

name_input = gets.chomp
name = name_input == 'Drake Cula' || name_input == 'Tu Fang'

Avoid using ternary statements; while they are shorter, they are harder to read